docs/implementation-references/getting-started-guide.md

```diff
@@ -159,27 +213,40 @@ Expect the following line on successful DI completion.
 DI complete, GUID is <guid>
 ```
 
-***NOTE:***
+**After completion of DI, the FDO credentials are stored into `credential.bin` file. The credentials file include `rvinfo` from manufacturer, which is later used by device to contact RV server, once it is powered on at the client side. Finally, the initialized device is boxed and sold to customers.**
```

Comment:
credential file "includes"
Instead of saying finally device is boxed, can we say the device is then boxed and sold to customers?

Reply:
Changed.